src/contour/display/TerminalWidget.cpp

```diff
@@ -684,6 +684,11 @@ void TerminalWidget::paint()
 
         terminal().tick(steady_clock::now());
         _renderer->render(terminal(), _renderingPressure);
+        if(_doDumpState)
```

Comment:
```suggestion
        if (_doDumpState)
```
Didn't clang-format check catch this? hmmm 🤔